src/docs/release/breaking-changes/index.md

```diff
@@ -35,6 +36,7 @@ The following guides (in alphabetical order) are available:
 [Annotations return local position relative to object]: /docs/release/breaking-changes/annotations-return-local-position-relative-to-object
 [breaking change policy]: /docs/resources/compatibility
 [Container color optimization]: /docs/release/breaking-changes/container-color
+[CupertinoTabBar requires Localizations parent]: /docs/release/breaking-changes/cupertino-tab-bar-localizations.md
```

Comment:
Delete the ".md" from the end. The generated file is not markdown, it's html.  Thx!